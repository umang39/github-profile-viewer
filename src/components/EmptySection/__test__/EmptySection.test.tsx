import React from "react";
import { EmptySection } from "@components/index";
import { cleanup } from "@testing-library/react";
import renderer from "react-test-renderer";
import { render, screen } from "@testing-library/react";
afterEach(cleanup);
it("error component for bio section running successfully", () => {
  render(<EmptySection title="bio" />);
  const text = screen.queryByText(/bio/i)?.innerHTML;
  expect(text).toBe("Oops ! You have no bio");
});
it("error component for blog section running successfully", () => {
  render(<EmptySection title="blog" />);
  const text = screen.queryByText(/blog/i)?.innerHTML;
  expect(text).toBe("Oops ! You have no blog");
});
it("matches snapshot", () => {
  const tree = renderer.create(<EmptySection title="bio" />).toJSON();
  expect(tree).toMatchSnapshot();
});
it("matches snapshot", () => {
  const tree = renderer.create(<EmptySection title="blog" />).toJSON();
  expect(tree).toMatchSnapshot();
});
