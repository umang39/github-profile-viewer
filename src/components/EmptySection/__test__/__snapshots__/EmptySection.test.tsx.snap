// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<h2
  className="MuiTypography-root MuiTypography-h2 makeStyles-text-3 css-cq2q8w-MuiTypography-root"
>
  Oops ! You have no 
  bio
</h2>
`;

exports[`matches snapshot 2`] = `
<h2
  className="MuiTypography-root MuiTypography-h2 makeStyles-text-3 css-cq2q8w-MuiTypography-root"
>
  Oops ! You have no 
  blog
</h2>
`;
